lib/Zonemaster/CLI.pm

```diff
@@ -601,6 +603,8 @@ sub add_fake_delegation {
             exit( 1 );
         }
 
+        $name =~ s/\.$// unless $name eq '.';;
```

Comment:
typo: two semi-colons

Reply:
To resolve zonemaster/zonemaster-engine#1055 the code must remove multiple trailing dots. Cf https://github.com/zonemaster/zonemaster-backend/pull/983#discussion_r862703867

Reply:
> typo: two semi-colons

fixed